README.md

```diff
@@ -1,34 +1,50 @@
 # BlogRest
+This application is used to create and manage a blog.
+The technical task is described in [Technical Task][1]
 
-## Technical task
-* Main functionality:
-1. User registration.
-2. Authorization.
-3. Adding an article.
-4. Editing an article.
-5. Deleting an article.
-6. Editing user.
+## Necessary tools
+* [Maven][2]
 
-* Additional functionality:
-1. Adding / removing a article like.
-2. Adding / removing comments on the article.
-3. Add user to subscriptions.
-4. Search.
+## Getting started
+Clone this repo to your local machine using:
+```
+git clone https://github.com/JavaBraveWarriors/BlogRest.git
+```
+To run the application on the embedded Jetty server, go to the project root and execute the following commands:
+```
+mvn -Pdev clean install 
```

Comment:
If some specific profiles is in use then it means that this configuration is for local development. So, it should be mentioned in README as well. Cause usually I just execute mvn clean build and that's it.